Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -270,4 +278,45 @@ protected function getFixersFromConfig($fileName, $config)
 
         return $fixers;
     }
+
+    /**
+     * Parses the '--REQUIREMENTS--' block of a '.test' file and determines requirements.
+     *
+     * @param string $fileName
+     * @param string $config
+     *
+     * @return array
+     */
+    protected function getRequirementsFromConfig($fileName, $config)
+    {
+        $requirements = array('hhvm' => true);
```

Comment:
I set default value for `hhvm` to avoid `isset` condition here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1523/files#diff-f387a0305e98fb7de24f5f8699ec1ec6R147